scripts/test.sh

```diff
@@ -81,26 +102,118 @@ start_packager() {
 }
 
 xctest() {
-  local dest="$(xcrun simctl list devices | grep -v unavailable | grep -m 1 -o '[0-9A-F\-]\{36\}')"
+  # note: the DEVELOPER_DIR and IOS_SIM_DEVICE sections sould be kept in sync with setup_ios_simulator
```

Comment:
What does "kept in sync" mean? They shouldn't be the same value, obviously. Does this refer to different versions of Xcode including different versions of available devices? If so, this could be reworded to say something along the lines of "This script may need to be updated to use available iOS Simulator devices if the Xcode version used changes.".